webapp/src/components/expanded_view/component.tsx

```diff
@@ -836,54 +960,55 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
         const isHost = this.props.callHostID === this.props.currentUserID;
         const isRecording = isHost && this.props.callRecording && this.props.callRecording.init_at > 0 && !this.props.callRecording.end_at && !this.props.callRecording.err;
         const recordTooltipText = isRecording ? formatMessage({defaultMessage: 'Stop recording'}) : formatMessage({defaultMessage: 'Record call'});
-        const RecordIcon = isRecording ? RecordSquareOutlineIcon : RecordCircleOutlineIcon;
+        const RecordIcon = isRecording ? RecordSquareIcon : RecordCircleIcon;
+        const ShareIcon = isSharing ? UnshareScreenIcon : ShareScreenIcon;
 
         return (
             <div
                 ref={this.expandedRootRef}
                 id='calls-expanded-view'
-                style={globalRhsSupported ? styles.root : {
-                    ...styles.root,
+                style={globalRhsSupported ? this.style.root : {
+                    ...this.style.root,
                     position: 'absolute',
                     top: 0,
                     left: 0,
                 }}
             >
-                <div style={styles.main}>
+                <div style={this.style.main}>
                     {this.renderAlertBanner()}
 
-                    <div style={styles.topContainer}>
-                        <div style={styles.topLeftContainer}>
-                            {this.renderRecordingBadge()}
-                            <CallDuration
-                                style={{margin: '4px'}}
-                                startAt={this.props.callStartAt}
+                    <div style={this.style.topContainer}>
+                        {this.renderRecordingBadge()}
+                        <CallDuration
+                            style={{margin: '4px'}}
+                            startAt={this.props.callStartAt}
+                        />
+                        <span style={{margin: '4px'}}>{untranslatable('•')}</span>
```

Comment:
Just curious, do we need the `untranslatable` here?

Reply:
How do you translate the dot? :p `untranslatable` is really just a way to add exceptions to the linter rules otherwise it will complain there are untranslated strings (not using formatMessage and the like).

Reply:
It complains? Interesting. I thought we could still use some raw strings, like blank spaces, dots, etc. Anyway, no matter, I was just curious.

Reply:
We can't since I added the formatjs linter rule to avoid missing translations. It will complain about any string without translation unless they are excluded.